src/test/enhetstester/kotlin/no/nav/familie/ks/sak/kjerne/behandling/BehandlingServiceTest.kt

```diff
@@ -0,0 +1,354 @@
+import no.nav.familie.ks.sak.kjerne.behandling.domene.BehandlingStatus
+import no.nav.familie.ks.sak.kjerne.behandling.domene.BehandlingType
+import no.nav.familie.ks.sak.kjerne.behandling.domene.BehandlingÅrsak
+import no.nav.familie.ks.sak.kjerne.behandling.steg.BehandlingSteg
+import no.nav.familie.ks.sak.kjerne.fagsak.domene.FagsakRepository
+import no.nav.familie.ks.sak.kjerne.logg.LoggService
+import no.nav.familie.ks.sak.kjerne.personident.PersonidentService
+import no.nav.familie.ks.sak.kjerne.vedtak.VedtakService
+import no.nav.familie.prosessering.domene.TaskRepository
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.EnumSource
+import java.time.LocalDate
+
+@ExtendWith(MockKExtension::class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class BehandlingServiceTest {
+    @MockK
+    private lateinit var personidentService: PersonidentService
+
+    @MockK
+    private lateinit var arbeidsfordelingService: ArbeidsfordelingService
+
+    @MockK
+    private lateinit var vedtakService: VedtakService
+
+    @MockK
+    private lateinit var loggService: LoggService
+
+    @MockK
+    private lateinit var fagsakRepository: FagsakRepository
+
+    @MockK
+    private lateinit var behandlingRepository: BehandlingRepository
+
+    @MockK
+    private lateinit var taskRepository: TaskRepository
+
+    @InjectMockKs
+    private lateinit var behandlingService: BehandlingService
+
+    private val søker = randomAktør()
+    val søkersIdent = søker.personidenter.first { personIdent -> personIdent.aktiv }.fødselsnummer
+    private val fagsak = lagFagsak(aktør = søker)
+    private val behandling = lagBehandling(
+        fagsak,
+        opprettetÅrsak = BehandlingÅrsak.SØKNAD
+    )
+
+    @BeforeEach
+    fun beforeEach() {
+        clearAllMocks()
+        every { personidentService.hentAktør(any()) } returns søker
+        every { fagsakRepository.finnFagsakForAktør(søker) } returns fagsak
+        every { behandlingRepository.findByFagsakAndAktiv(fagsak.id) } returns behandling.copy(
+            status = BehandlingStatus.AVSLUTTET,
+            aktiv = true
+        )
+        every { behandlingRepository.hentBehandling(any()) } returns behandling
+        every { behandlingRepository.finnBehandlinger(fagsak.id) } returns emptyList()
+        every { behandlingRepository.saveAndFlush(any()) } returns behandling
+        every { behandlingRepository.save(any()) } returns behandling
+        every { arbeidsfordelingService.fastsettBehandledeEnhet(any(), any()) } returns Unit
```

Comment:
```suggestion
        every { arbeidsfordelingService.fastsettBehandledeEnhet(any(), any()) } just runs
```
Man kan også bruke `just runs` istedenfor å returnere unit. Synes det gir litt mer mening semantisk.
Men er jo smak og behag 😛

Reply:
```suggestion
        justRun { arbeidsfordelingService.fastsettBehandledeEnhet(any(), any()) } 
```
Eventuelt dette^

Reply:
Ah, det visste jeg ikke jeg kunne gjøre! Fikser det! Takker og bukker 🙏